sw/device/silicon_creator/lib/drivers/pinmux.c

```diff
@@ -47,32 +30,60 @@ typedef struct pinmux_output {
 } pinmux_output_t;
 
 /**
- * MIO pad to peripheral output mappings.
+ * UART RX pin.
  */
-static const pinmux_output_t kPinmuxOutputs[] = {
-    /**
-     * UART
-     */
-    {.pad = kTopEarlgreyPinmuxMioOutIoc4,
-     .periph = kTopEarlgreyPinmuxOutselUart0Tx},
+static const pinmux_input_t kInputUart = {
+    .periph = kTopEarlgreyPinmuxPeripheralInUart0Rx,
+    .pad = kTopEarlgreyPinmuxInselIoc3,
 };
 
+/**
+ * Bootstrap pin.
```

Comment:
nit: Are we planning to enable support in the mask ROM to be able to decode all the bootstrap pins? Can we add a note to clarify?

Reply:
Captured in #11934, added TODO